HW03/vitaly_kadyrko/vitaly_kadyrko_hw_03_t_02.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task2(log)
+    output_array = []
+    good_data_array = log.split("\n").select { |x| x.length in 65..100 }
+    [] if good_data_array.size.zero?
```

Comment:
this line don't do anything 
```suggestion
    [] if good_data_array.size.zero?
```